src/templates/tag.js

```diff
@@ -117,7 +118,7 @@ const Tag = props => {
                             <AuthorByline>
                                 <SyledAuthorImage image={author_image} />
                                 <AuthorName>
-                                    <H2>{name}</H2>
+                                    <Title>{name}</Title>
```

Comment:
Code functionally looks great, we just want to add one more piece here similar to #735 with the page number. This time we also want the page number in the H1, did we want to add it in here as well?

We would like something like this screenshot here, let me know if this does not make sense.
![Screen Shot 2021-03-24 at 4 05 13 PM](https://user-images.githubusercontent.com/9064401/112376448-dc22b580-8cba-11eb-8cf0-806c3353a2bb.png)

Reply:
Thanks for help. I have added pagination logic for h1 title, please check new changes.